manoseimas/mps_v2/json_urls.py

```diff
@@ -14,4 +14,6 @@
     url(r'^discussion/(?P<statement_id>\d+)/?$',
         views.mp_discussion_json, name='mp_discussion_json'),
     url(r'^suggesters/?$', views.suggesters_json, name='suggesters_json'),
+    url(r'^resolutions/(?P<suggester_slug>.+)/?$',
```

Comment:
Hm, I see older code has `url(r'^statements/(?P<mp_slug>.+)/$',`.

I hate that Django uses regular expressions for URL patterns.

I still prefer `(?P<...>[^/]+)/?$`, but `(?P<...>.+)/$` would also be acceptable.

I need to read about Django URL patterns and automated `/` appending and related topics.